fluent-plugin-detect-exceptions.gemspec

```diff
@@ -11,7 +11,7 @@ eos
   gem.homepage      = \
     'https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions'
   gem.license       = 'Apache-2.0'
-  gem.version       = '0.0.1'
+  gem.version       = '0.0.2'
```

Comment:
I bumped the version number since this change adds new functionality, assuming that this change goes in before the change regarding the max_bytes parameter.